src/common/tiles/PersonTile/index.js

```diff
@@ -1,16 +1,15 @@
 import { imageBaseUrl } from "../../../ApiValue";
-import { Character, Container, Name, ProfileImage } from "./styled";
+import { Container, ProfileImage, Name, Info } from "./styled";
 import noPicture from "./noPicture.png";
 
-export const PersonTile = ({ id, profile_path, name, character }) => {
+export const PersonTile = ({/*  id, */ profile_path, name, large, character }) => {
   return (
-    <Container key={id} id={id}>
+    <Container /* key={id} id={id} */>
       <ProfileImage
         src={profile_path ? `${imageBaseUrl}/w342${profile_path}` : noPicture}
-        alt={`portrait of ${name}`}
-      ></ProfileImage>
+        alt={`portrait of ${name}`}/> 
       <Name>{name}</Name>
-      <Character>{character}</Character>
+      <Info big={large}>{character}</Info>
```

Comment:
nie za bardzo wiem o co chodzi z tymi big={large}, mogę prosić o wytłumaczenie :) ?

Reply:
Pewnie, big jest to props opisany w stylach csss. Potem jest  określony na elemencie jako large - `big={large}` i potem zostaje nadany large w strukturze np.  w MoviePage. Dzięki temu kafelki z  Poeople nie mają tych samych styli co w Cast i Crew, choć to bazowo ten sam element. 
Mam nadzieję że nie pomyliłem żadnych pojęć i udało mi się to dobrze wyjaśnić xD